### PR TITLE
Add text to MIT Libraries logo

### LINKED ISSUE #68: Add text to MIT Libraries logo
Add hidden text "MIT Libraries" with the Libraries logo in the header and footer

### PR DESCRIPTION
**Pull request by PBruk**
Tagging @frrrances for code-review: resolves #68.

### CODE REVIEW

File: css/partials/_layout.scss [hunk 1]
```diff
@@ -129,6 +129,16 @@ html.no-js .js-hidden {
 	}
 }
 
+/* for screen readers */
+.hidden-text {
```
Comment (frrrances) [inline css/partials/_layout.scss]: can we change this class to ".sr" for screenreader?

**Comment by frrrances** [comment]
one small change if possible, and then 👍

File: header-home.php [hunk 2]
```diff
@@ -41,12 +41,14 @@
...
+			<span class="hidden-text">MIT Libraries</span>
```
Comment (frrrances) [inline header-home.php]: I think this needs the "sr" class instead of the "hidden-text" one.

File: header-home.php [hunk 3]
```diff
@@ -41,12 +41,14 @@
...
+			<a class="link-logo-mit" href="http://www.mit.edu" alt="Massaschusetts Institute of Technology logo"><svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" width="54" height="28" viewBox="0 0 54 28" enable-background="new 0 0 54 28" xml:space="preserve" class="logo-mit"><rect x="28.9" y="8.9" width="5.8" height="19.1" class="color"/><rect width="5.8" height="28"/><rect x="9.6" width="5.8" height="18.8"/><rect x="19.3" width="5.8" height="28"/><rect x="38.5" y="8.9" width="5.8" height="19.1"/><rect x="38.8" width="15.2" height="5.6"/><rect x="28.9" width="5.8" height="5.6"/></svg><span>MIT Libraries</span></a>
```
Comment (frrrances) [inline header-home.php]: I think this is the MIT Logo - not the libraries logo. Can you change the text to say MIT and add the "sr" class?

**Comment by frrrances** [comment]
A few changes needed. Can you also check the indenting?

**Comment by PBruk** [comment]
Sorry, which indenting?

File: header-home.php [hunk 4]
```diff
@@ -41,12 +41,14 @@
...
....611v2.043c-2.847-0.235-4.724 1.474-4.724 4.319v10.886h-2.01V36.517h2.01V38.794z"/><path d="M99.936 36.517h2.009V53.43h-2.009V36.517zM99.7 29.249h2.479v2.545H99.7V29.249z"/><path d="M106.765 45.393v1.306c0 2.344 0.637 5.459 3.818 5.459 2.512 0 3.752-1.607 3.719-3.885h2.109c-0.234 4.086-2.445 5.492-5.828 5.492 -2.914 0-5.827-1.172-5.827-6.296v-4.153c0-4.89 2.043-7.134 5.827-7.134 5.828 0 5.828 4.187 5.828 9.211H106.765zM114.401 43.784c0-3.751-0.603-5.994-3.818-5.994 -3.215 0-3.818 2.243-3.818 5.994H114.401z"/><path d="M123.038 53.765c-3.651 0-5.092-1.774-5.058-5.358h2.043c0 2.312 0.535 3.852 3.114 3.852 1.943 0 2.88-1.104 2.88-2.981 0-4.42-7.603-3.281-7.603-8.875 0-3.114 2.144-4.219 5.092-4.219 3.349 0 4.42 2.311 4.42 4.889h-1.977c-0.1-2.144-0.67-3.282-2.947-3.282 -1.439 0-2.578 0.938-2.578 2.444 0 4.221 7.603 3.082 7.603 8.742C128.027 52.157 126.186 53.765 123.038 53.765"/><path class="arch" fill-rule="evenodd" clip-rule="evenodd" d="M121.487 19.746c-33.822-10.8-82.05-10.8-115.805 0C36.39 3.176 90.661 3.176 121.487 19.746"/><path class="arch" fill-rule="evenodd" clip-rule="evenodd" d="M105.419 8.979c-24.436-7.803-59.282-7.803-83.67 0C43.936-2.993 83.147-2.993 105.419 8.979"/></svg>
```
Comment (frrrances) [inline header-home.php]: indenting here - the open a tag doesn't line up with the close a tag.

File: header.php [hunk 5]
```diff
@@ -38,7 +38,9 @@
...
....611v2.043c-2.847-0.235-4.724 1.474-4.724 4.319v10.886h-2.01V36.517h2.01V38.794z"/><path d="M99.936 36.517h2.009V53.43h-2.009V36.517zM99.7 29.249h2.479v2.545H99.7V29.249z"/><path d="M106.765 45.393v1.306c0 2.344 0.637 5.459 3.818 5.459 2.512 0 3.752-1.607 3.719-3.885h2.109c-0.234 4.086-2.445 5.492-5.828 5.492 -2.914 0-5.827-1.172-5.827-6.296v-4.153c0-4.89 2.043-7.134 5.827-7.134 5.828 0 5.828 4.187 5.828 9.211H106.765zM114.401 43.784c0-3.751-0.603-5.994-3.818-5.994 -3.215 0-3.818 2.243-3.818 5.994H114.401z"/><path d="M123.038 53.765c-3.651 0-5.092-1.774-5.058-5.358h2.043c0 2.312 0.535 3.852 3.114 3.852 1.943 0 2.88-1.104 2.88-2.981 0-4.42-7.603-3.281-7.603-8.875 0-3.114 2.144-4.219 5.092-4.219 3.349 0 4.42 2.311 4.42 4.889h-1.977c-0.1-2.144-0.67-3.282-2.947-3.282 -1.439 0-2.578 0.938-2.578 2.444 0 4.221 7.603 3.082 7.603 8.742C128.027 52.157 126.186 53.765 123.038 53.765"/><path class="arch" fill-rule="evenodd" clip-rule="evenodd" d="M121.487 19.746c-33.822-10.8-82.05-10.8-115.805 0C36.39 3.176 90.661 3.176 121.487 19.746"/><path class="arch" fill-rule="evenodd" clip-rule="evenodd" d="M105.419 8.979c-24.436-7.803-59.282-7.803-83.67 0C43.936-2.993 83.147-2.993 105.419 8.979"/></svg>
```
Comment (frrrances) [inline header.php]: indenting here - the open a tag doesn't line up with the close a tag.

File: header-home.php [hunk 6]
```diff
@@ -41,22 +41,26 @@
...
+			<a class="link-logo-mit" href="http://www.mit.edu" alt="Massaschusetts Institute of Technology logo"><svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" width="54" height="28" viewBox="0 0 54 28" enable-background="new 0 0 54 28" xml:space="preserve" class="logo-mit"><rect x="28.9" y="8.9" width="5.8" height="19.1" class="color"/><rect width="5.8" height="28"/><rect x="9.6" width="5.8" height="18.8"/><rect x="19.3" width="5.8" height="28"/><rect x="38.5" y="8.9" width="5.8" height="19.1"/><rect x="38.8" width="15.2" height="5.6"/><rect x="28.9" width="5.8" height="5.6"/></svg>
+				<span>MIT Logo</span>
```
Comment (frrrances) [inline header-home.php]: Does this need an sr class?

File: header.php [hunk 7]
```diff
@@ -38,22 +38,26 @@
...
+			<a class="link-logo-mit" href="http://www.mit.edu" alt="Massaschusetts Institute of Technology logo"><svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" width="54" height="28" viewBox="0 0 54 28" enable-background="new 0 0 54 28" xml:space="preserve" class="logo-mit"><rect x="28.9" y="8.9" width="5.8" height="19.1" class="color"/><rect width="5.8" height="28"/><rect x="9.6" width="5.8" height="18.8"/><rect x="19.3" width="5.8" height="28"/><rect x="38.5" y="8.9" width="5.8" height="19.1"/><rect x="38.8" width="15.2" height="5.6"/><rect x="28.9" width="5.8" height="5.6"/></svg>
+				<span>MIT Logo</span>
```
Comment (frrrances) [inline header.php]: Does this need an sr class?

**Comment by frrrances** [comment]
Lgtm 👍

**Pull request merged**